From 2e73945430a8408f671e662424cfbd6d830d95bf Mon Sep 17 00:00:00 2001
From: Patrick Allison <allison.122@osu.edu>
Date: Tue, 22 Oct 2024 17:21:53 +0000
Subject: [PATCH] disable nonexistent vt

---
 presets/90-systemd.preset | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/presets/90-systemd.preset b/presets/90-systemd.preset
index d26087445c..58905b8a59 100644
--- a/presets/90-systemd.preset
+++ b/presets/90-systemd.preset
@@ -15,7 +15,7 @@ enable remote-fs.target
 enable remote-cryptsetup.target
 enable machines.target
 
-enable getty@.service
+disable getty@.service
 enable systemd-timesyncd.service
 enable systemd-networkd.service
 enable systemd-resolved.service
