From 342f0038b00e51285826cb5b9ed69f53bcae53dd Mon Sep 17 00:00:00 2001
From: Patrick Allison <allison.122@osu.edu>
Date: Wed, 26 Feb 2025 16:39:33 +0000
Subject: [PATCH 1/1] add debugging to DA9062 because it's a disaster

---
 drivers/watchdog/da9062_wdt.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/watchdog/da9062_wdt.c b/drivers/watchdog/da9062_wdt.c
index 706fb09c2f24..c076fec1aef3 100644
--- a/drivers/watchdog/da9062_wdt.c
+++ b/drivers/watchdog/da9062_wdt.c
@@ -68,6 +68,8 @@ static int da9062_wdt_update_timeout_register(struct da9062_watchdog *wdt,
 {
 	struct da9062 *chip = wdt->hw;
 
+	dev_info(wdt->hw->dev, "Actually updating timeout register (val = %d)\n",
+		 regval);
 	regmap_update_bits(chip->regmap,
 				  DA9062AA_CONTROL_D,
 				  DA9062AA_TWDSCALE_MASK,
@@ -100,6 +102,8 @@ static int da9062_wdt_stop(struct watchdog_device *wdd)
 {
 	struct da9062_watchdog *wdt = watchdog_get_drvdata(wdd);
 	int ret;
+	
+	dev_info(wdt->hw->dev, "Stopping timeout register\n");
 
 	ret = regmap_update_bits(wdt->hw->regmap,
 				 DA9062AA_CONTROL_D,
-- 
2.25.1

